src/openforms/contrib/kvk/tests/base.py

```diff
@@ -12,7 +12,7 @@
 
 TEST_FILES = Path(__file__).parent.resolve() / "files"
 
-G1_ROOT = TEST_FILES / "Staat_der_Nederlanden_Private_Root_CA_-_G1.crt"
+G1_ROOT = TEST_FILES / "DigiCert_Global_Root_G2.crt"
```

Comment:
this root certificate appears to be in the browser trust stores and also in the requests/certifi root store, so in short - there is no reason (anymore) to explicitly allow list this.

I think that lines 30-34 and L43 can be removed. If `G1_ROOT` is otherwise unused, that can also be removed and the associated file.

For more context - the G1 root is deliberately a _private_ root certificate issued by the Dutch state. Certain companies (like KPN) have an intermediate certificate, signed by the G1 root. This means they are trusted and verified parties that can issue government-related certificates to servers/clients participating in (m)TLS traffic.

It used to be that the KVK test server was using a server certificate issued by such an intermediate, and thus, trusting the G1 root (which is typically not in trust stores and definitely not in `certifi`) causes the intermediates and then also the leaf certificates to be trusted, preventing those "pesky" verification failed errors.

So, the `G1_ROOT` as a name has certainly and important meaning and implications, and it should not be patched like this.